doc/Icetea.md

```diff
@@ -223,20 +223,21 @@ See [suite api](suite_api.md) for more info.
 
 ### console results
 ```
-  +--------------+---------+-------------+-------------+-----------+-----------+
-  | Testcase     | Verdict | Fail Reason | Skip Reason | platforms |  duration |
-  +--------------+---------+-------------+-------------+-----------+-----------+
-  | test_cmdline |   pass  |             |             |    K64F   | 10.950142 |
-  +--------------+---------+-------------+-------------+-----------+-----------+
-  +---------------+----------------+
-  |    Summary    |                |
-  +---------------+----------------+
-  | Final Verdict |      PASS      |
-  |     count     |       1        |
-  |    passrate   |    100.00 %    |
-  |      pass     |       1        |
-  |    Duration   | 0:00:10.950142 |
-  +---------------+----------------+
+    +--------------+---------+-------------+-------------+-----------+----------+---------+
+    | Testcase     | Verdict | Fail Reason | Skip Reason | Platforms | Duration | Retried |
+    +--------------+---------+-------------+-------------+-----------+----------+---------+
+    | test_cmdline |   pass  |             |             |  process  | 0.946598 |    No   |
```

Comment:
maybe retried column should show count of retries when it happens not just "Yes/No".

Reply:
well, then it should be called retries left or something else. I could do that if that's something we want.